source/index.html.md

```diff
@@ -68,36 +76,68 @@ curl -X POST "https://app.grooveapp.com/oauth/authorize"
+| client_secret | the client secret configured for your OAuth App                                              |
+
+**Note: Please avoid exposing your client secret**
 
 The returned response body should contain a `JSON` payload with the following key/value pairs
 
-  * `access_token` - the value represents the string to add to the `Authorization` header of future API requests
-  * `created_at` - the value represents the Unix timestamp when the access token was created
-  * `token_type` - the value will be `Bearer`
-  * `expires_in` - the value indicates the number of seconds until the access token will be invalid
-  * `refresh_token` - the value indicates the string to use to generate another access token
-  * `scope` - a space-separated string indicating the scopes that the access token can use
+| key           | value                                                                                        |
+|---------------|----------------------------------------------------------------------------------------------|
+| access_token  | the value represents the string to add to the `Authorization`  header of future API requests |
+| created_at    | the value represents the Unix timestamp when the access token was created                    |
+| token_type    | the value will be `Bearer`                                                                   |
+| expires_in    | the value indicates the number of seconds until the access token will be invalid             |
+| refresh_token | the value indicates the string to use to generate another access token                       |
+| scope         | a space-separated string indicating the scopes that the access token can use                 |
+
+## Making Authenticated API Calls
+
+Use the generated `access_token` value to make API requests on behalf of the authorized user in the `Authorization` header with the header value having the following format: `Authorization: Bearer {YOUR_ACCESS_TOKEN}`
+
+> A authenticated CURL may look like:
+
+```
+curl -X POST "https://app.grooveapp.com/api/public/..."
+  -H 'Content-Type: application/json'
+  -H 'Authorization: Bearer {YOUR_ACCESS_TOKEN}'
+  -d ...
+```
 
-* Use the generated `access_token` value to make API requests on behalf of the authorized user in the `Authorization` header with the header value having the following format - `Authorization: Bearer {YourAccessToken}`
 
 ## Access Token Expiration
 
-An access token expires 12 minutes after creation. The expiration time is specified by the `expires_in` field in the token creation response body.
+An access token expires 2 hours after creation. The expiration time is specified by the `expires_in` field in the token creation response body.
+
+In order to generate a new access token, use the refresh token, specified by the `refresh_token` field in the token creation response body, to make a `POST` request to `/oauth/token/refresh` with the refresh token.
```

Comment:
This was an error I made when I initially made the docs - feel free to change the error around refresh tokens here (and add an example) or I can do so in a follow-up PR.